### PR TITLE
New styling and first illustration

### PR DESCRIPTION
**Pull request by dburka**

### CODE REVIEW

File: src/app/page.tsx [hunk 1]
```diff
@@ -3,11 +3,15 @@ import { Footer } from '../components/Footer';
 import NewsletterForm from '../components/NewsletterForm';
 import { Map } from '../components/Map';
 import { Team } from '../components/Team';
+import Image from 'next/image';
 
 export default function Home() {
   return (
     <>
       <section className="left">
+        <div class="illustration-home">
```
Comment (nburka) [inline src/app/page.tsx]: this needs to be `className` instead of `class`

Comment (nburka) [reply src/app/page.tsx]: it's a React thing… I know.

File: src/app/page.tsx [hunk 2]
```diff
@@ -3,11 +3,15 @@ import { Footer } from '../components/Footer';
...
+          <Image src="/images/illustration-directions.svg" width="80" height="80" alt="" />
```
Comment (nburka) [inline src/app/page.tsx]: can you add a real "alt" for accessibility?

**Review by nburka — CHANGES_REQUESTED** [review]

File: src/app/page.tsx (hunk 2 above)
Comment (dburka) [reply src/app/page.tsx]: Done.

File: src/app/page.tsx (hunk 1 above)
Comment (dburka) [reply src/app/page.tsx]: Fixed

**Pull request merged**